addons/website_forum/static/src/js/website_forum.js

```diff
@@ -255,10 +256,10 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             if (forumID) {
                 notifOptions.messageIsHtml = true;
                 const linkLabel = _.escape(_t("Read the guidelines to know how to gain karma."));
-                notifOptions.message = `
+                notifOptions.message = Markup(`
                     ${_.escape(notifOptions.message)}<br/>
                     <a class="alert-link" href="/forum/${forumID}/faq">${linkLabel}</a>
-                `;
+                `);
```

Comment:
```suggestion
                const linkLabel = _t("Read the guidelines to know how to gain karma.");
                notifOptions.message = Markup`
                    ${notifOptions.message}<br/>
                    <a class="alert-link" href="/forum/${forumID}/faq">${linkLabel}</a>
                `;
```
from [the Markup code](https://github.com/odoo/odoo/blob/c169c9d7db098e2c09fe08f76fcbe7d5cb7b82b4/addons/web/static/src/legacy/js/core/utils.js#L270-L272), you can drop the parenthesis and then what is not a markup variable (3 variables here I assume) will be escaped. This should have exactly the same effect as your current code (only `forumID` will be escaped too but it should be done anyway).

Reply:
Updated :+1: Thanks!